{{cookiecutter.project_slug}}/.gitlab/gitlab-ci.yml

```diff
@@ -6,7 +6,8 @@ variables:
   # have some prefix so it may help for garbage collecting in the registry
   SC_CI_PROJECT_PATH_NAME: ci/$CI_PROJECT_PATH_SLUG/$CI_COMMIT_REF_SLUG
   SC_CI_TEST_IMAGE_PREFIX: $SC_CI_TESTING_REGISTRY/ci/$CI_PROJECT_PATH_SLUG/$CI_COMMIT_REF_SLUG
-  SC_CI_{{ cookiecutter.project_package_name.upper() }}_LOCATION: services/{{ cookiecutter.project_slug }}
+  SC_CI_{{ cookiecutter.project_package_name.upper().replace("-", "_").replace("-", "_") }}_LOCATION: services/{{ cookiecutter.project_slug }}
```

Comment:
is there not some specific variable for this? such as this _slug one?

Reply:
Yep, you're right. Actually `project_package_name` already ensure that there are "_" instead of spaces or "-". I reverted to the previous version.